### PR TITLE
feat(api): Add event-ownership endpoint

### PR DESCRIPTION
**Pull request by mattrobenolt**
Need to just add some tests.

cc @MaxBittker Let's make sure this is what we'll need to consume from the UI.

### CODE REVIEW

File: src/sentry/models/projectownership.py [hunk 1]
```diff
@@ -25,3 +26,59 @@ class Meta:
...
+
+
+def make_actor(owner, project_id):
+    """ Convert an Owner object into an Actor """
+    from sentry.api.fields.actor import Actor
+    from sentry.models import User, Team
+
+    if owner.type == 'user':
+        try:
+            # TODO(mattrobenolt): Needs to be scoped within org membership
```
Comment (mattrobenolt) [inline src/sentry/models/projectownership.py]: Reminding myself for tomorrow.

File: src/sentry/models/projectownership.py [hunk 2]
```diff
@@ -25,3 +26,59 @@ class Meta:
...
+    """ Convert an Owner object into an Actor """
+    from sentry.api.fields.actor import Actor
+    from sentry.models import User, Team
+
+    if owner.type == 'user':
+        try:
+            user_id = User.objects.filter(
+                email__iexact=owner.identifier,
+                is_active=True,
+                sentry_orgmember_set__organizationmemberteam__team__projectteam__project_id=project_id,
```
Comment (mattrobenolt) [inline src/sentry/models/projectownership.py]: ORM god

The gist of this is "give me the user who matches this email address and who has access to this project through team and org membership relationships.

**Review by evanpurkhiser — APPROVED** [review]
Looking good to me, could just use a test or two. 📜

**Comment by mattrobenolt** [comment]
Added a few tests to cover the `ProjectOwnership.get_owners` bit since that's the most critical and will be used elsewhere.

File: src/sentry/api/fields/actor.py [hunk 3]
```diff
@@ -76,6 +79,9 @@ def resolve_dict(cls, actor_dict):
             for key, value in actor_dict.items()
         }
 
+    def __eq__(self, other):
```
Comment (mattrobenolt) [inline src/sentry/api/fields/actor.py]: Thanks @MaxBittker

File: tests/sentry/models/test_projectownership.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+class ProjectOwnershipTestCase(TestCase):
+    def test_get_owners_default(self):
+        assert ProjectOwnership.get_owners(self.project.id, {}) == ProjectOwnership.Everyone
+
+    def test_get_owners_basic(self):
+        ProjectOwnership.objects.create(
+            project_id=self.project.id,
+            schema=dump_schema([
+                Rule(Matcher('path', '*.py'), [Owner('user', self.user.email)]),
```
Comment (EvanPurkhiser) [inline tests/sentry/models/test_projectownership.py]: Maybe one more rule for matching to a team?

**Pull request merged**